packages/steiger/src/shared/globs/create-filter-according-to-globs.ts

```diff
@@ -13,29 +13,29 @@ export function createFilterAccordingToGlobs({ inclusions, exclusions }: ApplyGl
   const thereAreInclusions = Array.isArray(inclusions)
   const thereAreExclusions = Array.isArray(exclusions)
   const inclusionsEmpty = thereAreInclusions && inclusions.length === 0
+  const picomatchOptions = { posixSlashes: true }
 
-  function filterAccordingToGlobs(path: string) {
-    const matchesInclusionPatterns =
-      !thereAreInclusions || inclusions.some((pattern) => micromatch.isMatch(path, pattern))
-    let isIgnored = false
+  const isIncluded = thereAreInclusions ? picomatch(inclusions, picomatchOptions) : () => true
+
+  const positiveExclusionPatterns =
+    (thereAreExclusions && exclusions.filter((pattern) => !isNegatedGlob(pattern))) || []
+  const negativeExclusionPatterns =
```

Comment:
suggestion (non-blocking): since "positive exclusion" and "negative exclusion" aren't commonly understood terms, hence they require some time to understand, perhaps we could instead have "positive exclusion" and "re-inclusion"?

```suggestion
  const reInclusionPatterns =
```

Reply:
Agree. Fixed